.babelrc

```diff
@@ -1,4 +1,4 @@
 {
   "presets": ["@babel/preset-env", "@babel/preset-react"],
-  "plugins": ["babel-plugin-styled-components"]
+  "plugins": ["babel-plugin-styled-components", "@babel/plugin-proposal-optional-chaining"]
```

Comment:
Have you added that to package.json?

Reply:
No, we don't need it. All we need is a babel-eslint parser ;D

Reply:
But don't you need to install this plugin? Or is it already installed?

Reply:
So remove that entry